docs/user_guide/installation.rst

```diff
@@ -0,0 +1,39 @@
+Installation
+============
+
+container (preferred)
+---------------------
+
+We recommend using the Neurodocker Docker image, which can be access through
+Docker or Singularity.
+
+.. code-block:: bash
+
+    docker run --rm repronim/neurodocker:0.7.0 --help
+
+.. code-block:: bash
+
+    singularity run docker://repronim/neurodocker:0.7.0 --help
+
+pip
+---
+
+Neurodocker can also be installed with :code:`pip`. This is useful if you want to use
```

Comment:
minimal version of python should be added

Reply:
Agree